scripts/plot_benchmark.py

```diff
@@ -177,7 +190,7 @@ def plot_benchmark(
     options: dict,
     colors: dict,
     model_col: str = "Open-TYNDP",
-    rfc_col: str = "TYNDP 2024",
+    rfc_col: list[str] = ["TYNDP 2024", "TYNDP 2024 Vis Plfm"],
```

Comment:
Only if my peeve from above is accepted 😄 
```suggestion
    rfc_col: list[str] = ["TYNDP 2024", "TYNDP 2024 Vis Pltfm"],
```